argus/scenarios/service_mock.py

```diff
@@ -88,11 +89,47 @@ def stop_me(self):  # pylint: disable=no-self-use
         cherrypy.engine.exit()
 
 
-class EC2MetadataServiceApp(BaseServiceApp):
-    pass
+class MetadataServiceAppMixin(object):
+    """Common metadata resources."""
+
+    def instance_id(self, *_):
```

Comment:
Do you actually need an extra argument? If so, it's better to specify it and use a default value. *_ is just hideous.